asylum-connect/catalog-api
# Built middleware for JWT verification and added to routes.

## Description

Added the verifyToken.js middleware which expects a JSON Web Token to be provided in the HTTP headers of the request. Also added a small function to the utils file for generating a JWT.

- Asana ticket: https://app.asana.com/0/1148774074088494/1164471184183571

## PR Checklist




## How to Test

1. Run locally.
2. Use Postman to submit a POST request to http://localhost:8080/v1/organizations.
3. Observe that without a x-json-web-token header and a valid JWT that the response is an error code.
